packages/app/src/locale/index.ts

```diff
@@ -1,17 +1,19 @@
-import localefile from './APP_LOCALE';
+import nl from './nl.json';
+import en from './en.json';
 
-export type Locale = typeof localefile;
+export type NLLocale = typeof nl;
+export type ENLocale = typeof en;
 
 export type Languages = {
-  nl: Locale;
-  en: Locale;
+  nl: NLLocale;
+  en: ENLocale;
```

Comment:
```suggestion
  nl: NlLocale;
  en: EnLocale;
```
As mentioned earlier, I prefer to camel-case acronyms like regular words to keep things consistent